Sources/TuistKit/Cache/CacheController.swift

```diff
@@ -103,7 +104,7 @@ final class CacheController: CacheControlling {
         self.focusServiceProjectGeneratorFactory = focusServiceProjectGeneratorFactory
     }
 
-    func cache(path: AbsolutePath) throws {
+    func cache(path: AbsolutePath, targetsToFilter: [String]?) throws {
```

Comment:
I think checking for whether the array is empty is maybe more comprehensible ... or what do you think? It might make the logic more straightforward

Reply:
I felt that the optionality here it make the logic type safer
i.e. if not exist we should not filter, if exist we can filter (instead of excluding the empty array)
Anyhow, if we used your suggested approach in other part of the project I would adapt for consistency. Shall we? 

(Btw the logic itself won't change much, if let vs if .isEmpty)

Reply:
I don't think there is a consensus on which version to use in this project (after looking over the code), I *believe* that empty array might be more common but either way it's fine. Let's keep it as is 🙂

Reply:
Changed anyway :) It was a 1 sec change anyway.